README.md

```diff
@@ -110,6 +110,26 @@ For initial microbenchmark performance, we add 10 fields up with spark, data siz
 
 ![Performance](./docs/image/performance.png)
```

Comment:
can you please also remove the outdated perf data?

Reply:
Done